src/Sulu/Bundle/AdminBundle/DependencyInjection/Compiler/AddAdminPass.php

```diff
@@ -28,11 +28,24 @@ public function process(ContainerBuilder $container)
     {
         $pool = $container->getDefinition('sulu_admin.admin_pool');
 
-        $taggedServices = $container->findTaggedServiceIds(self::ADMIN_TAG);
+        $adminServiceDefinitions = [];
+        foreach ($container->findTaggedServiceIds(self::ADMIN_TAG) as $id => $tags) {
+            $serviceDefinition = $container->getDefinition($id);
 
-        foreach ($taggedServices as $id => $attributes) {
-            $admin = $container->getDefinition($id);
-            $pool->addMethodCall('addAdmin', [$admin]);
+            $class = $container->getParameterBag()->resolveValue($serviceDefinition->getClass());
```

Comment:
neccessary to be sure that class parameters are resolved (example `%sulu_media.admin.class%`)